Android/app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,24 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+	<FrameLayout
+		android:id="@+id/flFragment"
+		android:layout_width="match_parent"
+		android:layout_height="653dp"
```

Comment:
Hard-coding heights is generally a frowned upon practice. This is fine for our projects but if we have time I'd enjoy showing you how to take advantage of constraint layout

Reply:
sure that would be incredible